WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -414,6 +414,26 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
         headerView.onGravatarPress = { [weak self] in
             self?.presentGravatarPicker()
         }
+        headerView.onDroppedImage = { [weak self] image in
+            let imageCropViewController = ImageCropViewController.init(image: image!)
```

Comment:
In Swift, you don't need `.init` – just `ImageCropViewController(image: image!)` will do.